src/test/kotlin/no/nav/fo/veilarbregistrering/oppfolging/adapter/OppfolgingClientTest.kt

```diff
@@ -114,6 +116,30 @@ internal class OppfolgingClientTest(private val mockServer: ClientAndServer) {
         assertThat(exception.aktiverBrukerFeil).isEqualTo(AktiverBrukerFeil.BRUKER_MANGLER_ARBEIDSTILLATELSE)
     }
 
+    @Test
+    fun `skal kaste riktig feil ved manglende oppholdstillatelse for reaktivering`() {
+        mockServer.`when`(HttpRequest.request().withMethod("POST").withPath("/oppfolging/reaktiverbruker")).respond(
+            HttpResponse.response().withBody(FileToJson.toJson("/oppfolging/manglerOppholdstillatelse.json"))
+                .withStatusCode(403)
+        )
+        val exception: AktiverBrukerException = assertThrows {
+            oppfolgingClient.reaktiverBruker(FNR)
+        }
+        assertThat(exception.aktiverBrukerFeil).isEqualTo(AktiverBrukerFeil.BRUKER_MANGLER_ARBEIDSTILLATELSE)
+    }
+
+    @Test
+    fun `skal kaste riktig feil ved manglende oppholdstillatelse for sykmeldt-aktivering`() {
```

Comment:
Burde testnavnet ha navngitt hva som var riktig feil? For å fungere bedre som doc

Reply:
[eacba28](https://github.com/navikt/veilarbregistrering/pull/638/commits/eacba282940f8f37d27ad2dbb6d74c8ab6f835dc)